File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.py
s1, s2 = input(), input()
cnt1, cnt2 = [0]*26, [0]*26
for c in s1:
    cnt1[ord(c) - ord('a')] += 1
for c in s2:
    cnt2[ord(c) - ord('a')] += 1
ret = 0
for i in range(26):
    ret += abs(cnt1[i] - cnt2[i])
print(ret)

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        var in = new BufferedReader(new InputStreamReader(System.in));
        var out = new PrintWriter(System.out);

        String s1 = in.readLine();
        String s2 = in.readLine();
        int[] cnt1 = new int[26];
        int[] cnt2 = new int[26];
        for(int i = 0; i < s1.length(); i++) cnt1[s1.charAt(i) - 'a']++;
        for(int i = 0; i < s2.length(); i++) cnt2[s2.charAt(i) - 'a']++;
        int ret = 0;
        for(int i = 0; i< 26; i++){
            ret += Math.abs(cnt1[i] - cnt2[i]);
        }
        out.println(ret);
        
        out.flush();
        out.close();
    }
}
/*

*/